distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -42,10 +42,10 @@ org.codice.ddf.system.port=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
 org.codice.ddf.system.rootContext=/services
 # external properties are useful when DDF is behind a proxy and
 # the URL a user sees is different than the local url DDF uses
-org.codice.ddf.external.protocol=_DO_NOT_EXPAND_${org.codice.ddf.system.protocol}
-org.codice.ddf.external.hostname=_DO_NOT_EXPAND_${org.codice.ddf.system.hostname}
-org.codice.ddf.external.httpsPort=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
-org.codice.ddf.external.httpPort=_DO_NOT_EXPAND_${org.codice.ddf.system.httpPort}
+org.codice.ddf.external.protocol=https://
```

Comment:
Is there any documentation that needs to be updated with this? I though we outlined somewhere what fields to change in this file when updating port/host info. If so we need to update it to include the external fields as well.

Reply:
Good catch, let me double check the docs 👍

Reply:
It looks like the docs already had these values expanded so I don't think there's anything that needs to be updated here.